projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/date-time-field/date-time-field.component.html

```diff
@@ -1,5 +1,5 @@
 <span class="token-picker-field-box" *ngIf="!isDynamic else dynamic">
-  <mat-form-field *ngIf="!showTokenTextBox" title="{{lf_field_info.name}}" appearance="outline" class="lf-field">
+  <mat-form-field backdropClass="lf-datetime-picker" class="lf-datetime-picker" *ngIf="!showTokenTextBox" title="{{lf_field_info.name}}" appearance="outline" class="lf-field">
```

Comment:
I don't think the class on the form field should be the same as the overlay. Unless there's a reason for that?